src/server.c

```diff
@@ -4071,6 +4071,19 @@ int processCommand(client *c) {
         return C_OK;
     }
 
+    if ((c->cmd->flags & CMD_INTERNAL) && !((c->flags & CLIENT_INTERNAL) || mustObeyClient(c))) {
+        sds args = sdsempty();
+        int i;
+        for (i=1; i < c->argc && sdslen(args) < 128; i++)
```

Comment:
```suggestion
        for (int i = 1; i < c->argc && sdslen(args) < 128; i++)
```
Maybe more blank space will be better

Reply:
NP, modified.